channels-rke2.yaml

```diff
@@ -35,6 +35,8 @@ releases:
         - rke2-ingress-nginx
         - rke2-kube-proxy
```

Comment:
Should this be removed as an option if we are only allowing to enable the static pod?

Reply:
hmmmm, okay that makes sense, removing